donate/thunderbird/templates/payment/thank_you.html

```diff
@@ -10,21 +10,23 @@
 {% block buttons %}
     {% trans "I donated to @mozthunderbird today to #freetheinbox. Join me to support communication privacy." as twitter_text context "Used as a tweet" %}
     {% trans "I donated to Thunderbird today" as email_subject context "Email subject line" %}
+    {% get_current_language as LANGUAGE_CODE %}
+    {% with page_link="https://donate.mozilla.org/"|add:LANGUAGE_CODE|add:"/thunderbird/"%}
```

Comment:
We shouldn’t redirect to the old stack, as the locale lists can (and will) differ.

Also since Thunderbird will have its own domain, I’m not sure adding `/thunderbird` still makes sense. I’m guessing we will redirect `https://donate.mozilla.org/thunderbird/` to the dedicated Thunderbird domain once we’re fully live. So we should be good doing the same as for Mozilla:

```suggestion
    {% with page_link=request.scheme|add:"://"|add:request.get_host|add:"/"|add:LANGUAGE_CODE %}
```

Reply:
For this PR I'm just fixing the existing locale bug. There's another [ticket](https://github.com/mozilla/donate-wagtail/issues/597) for updating the Thunderbird URL (which I will tackle after this PR gets merged). @TheoChevalier would you suggest that I do everything in this PR instead of seperating them into 2 PRs?

Reply:
@alanmoo do you know what share link we will end up using for Thunderbird? If so, maybe I can just update this PR to include the fix to save us PR filing & reviewing time.

Reply:
That's up to Ryan and Phillip, I'd say shoot them an email.

Reply:
k, I've sent an email to them. I'll file a separate PR update the links once I hear back from them.